### PR TITLE
Select the next driver by clicking on him

### LINKED ISSUE #4: Click on a person to select her
I would like to select the next person to be the driver by clicking on her name.

### PR DESCRIPTION
**Pull request by ajsaraujo**
Now it's possible to select the next driver by clicking on his name. 

Resolves #4 

https://user-images.githubusercontent.com/34658414/170150164-52e39c00-b9da-4aea-9af0-b282c20a436f.mp4

### CODE REVIEW

File: src/components/person.js [hunk 1]
```diff
@@ -3,23 +3,30 @@ import PeopleActions from '../actions/people_actions';
...
     var classes = ClassNames({
       active: this.props.isCurrent
     });
 
     return (
-      <li className={classes}>
-        <a href="#" className="remove" onClick={this.handleClick.bind(this)}>x</a>
+      <li className={classes} onClick={this.select.bind(this)}>
+        <a href="#" className="remove" onClick={event => {
+          event.stopPropagation()
```
Comment (ajsaraujo) [inline src/components/person.js:22]: We have to stop propagation, or else the click will trigger the `li`'s click event, thus calling the "select driver" action.

**Comment by zoeesilcock** [comment]
This looks great, thanks for the PR! I have tested it on my machine and I don't see any problems.

A minor improvement would be to set the cursor to `pointer` when hovering over the person list items to make it clear that they are now clickable. Please add that and then I'd say it's ready for merging.

**Comment by ajsaraujo** [comment]
Done! I also changed the background color on hover, so it's more clear which person you are selecting. I guess changing the cursor to pointer wouldn't be as nice, since the "person" elements have no margin between them, the cursor would not change at all and then the user would have no feedback.

**Comment by zoeesilcock** [comment]
Yeah, the change in color on hover makes it even better, good work! I'll merge this now.

**Pull request merged**